test/fixtures/test-runner/output/hooks.snapshot

```diff
@@ -181,7 +181,7 @@ not ok 4 - beforeEach throws
         *
         *
         *
-        *
+        async Promise.all (index 0)
```

Comment:
Sorry, I don't really know what the test runner is doing. But why does this particular stack frame exist now? What's going on here?

Reply:
this diff is here since previously the `.out` files where generated manually, and now they are generated automatically.
so stack trace lines are detected using a regex and replaced with `*`.
these specific lines are not detected as stack trace lines and thus are not replaces which is ok (since what usually should be ignored is file paths and location in the file etc)